### ISSUE TITLE
Allow outdenting before enclosing parentheses

### ISSUE THREAD

**Issue by zweiter** [perltidy/perltidy#55]
As soon as a (complex, e.g. containing other data structures) list is too long to fit in its enclosing parentheses (with `-lp` and `-bbhb`, `-bbp`, `-bbsb`) it falls back to its default of not lining up the enclosing parens:
```pl
$object -> some_function (
                    'Some not too long a string',
                    {
                      label => ['something long that needs to be here and would be hard to shorten considerably sadly']
                    }
) ; # jumped to start of line

#<<<
$object -> some_function (
                    'Some not too long a string',
                    {
                      label => ['something long that needs to be here and would be hard to shorten considerably sadly']
                    }
                         ) ; # expected to stay here, although behind enclosing parens
#>>>
```

**Comment by shancock9** [comment]
This formats correctly in the development version after commit 872d4b4, the fix for issue git #54.  The problem was that the various -break-before- flags and the -lp flag were not being coordinated.  The next release to CPAN should be about the end of this month.

**Comment by zweiter** [comment]
That's great to hear! Thanks a million and have a nice weekend!
